gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -649,8 +672,11 @@ public TraceRequest getTraceRequest(Settings settings) {
           options.setDisablePcs(disablePcs.getSelection());
         }
 
-        if (config.getType() == TraceType.Perfetto) {
-          readPerfettoConfig(options);
+        if (isPerfetto(config)) {
+          int duration = frameCount.getSelection() * 1000;
+          // TODO: this isn't really unlimitted.
```

Comment:
Use the `TODO(username):` format?

Reply:
The username portion is optional and we have mostly not been using it in GAPID (small team and all).